bagisto/bagisto
# Fixed #6820 - Customer Group Price adding details in price section is…

## Issue Reference
#6820 

## Description
issue was fixed by modifying this file: bagisto\packages\webkul\Admin\src\Resources\views\catalog\products\accordians\customer-group-price.blade.php
removed the styling in line 6-8 to resolve issue.

before fixed:
![beforefixed](https://user-images.githubusercontent.com/76978538/194675310-ce06a7db-f645-41dc-9a3a-1a6e8a0f8162.jpg)

after fixed:
![issuefixed](https://user-images.githubusercontent.com/76978538/194675312-2b7a16f6-d39c-4322-a970-5accb8862771.jpg)


## How To Test This?
In admin
Create or Update a Product -> Price Section -> Add Customer Group price -> There we can see the UI Issue.

## Documentation
